.cursor/rules/PageFrame.md

```
@@ -0,0 +1,143 @@
+      "since": "5.0_3.3377A.20181128.161740",
+      "samples": [
+        "$p.parent().ibx_exam1.setValue(\"Example\");",
+        "$p.parent().scwin.getTopPageInfo();"
+      ]
+    }
+  }
+}
+
+```
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)